phpunit/web/Glpi/Inventory/RequestTest.php

```diff
@@ -464,4 +464,203 @@ public function testAuthBasicWithFakeCredential()
             $this->checkJsonResponse($response, '{"status":"error","message":"Access denied. Wrong login or password for basic authentication.","expiration":24}', 401);
         }
     }
+
+    public function testAuthOAuthClientCredentials()
+    {
+        /** @var mixed $DB */
+        global $DB;
+
+        // enable inventory with OAuth client credentials
+        $this->login();
+        $conf = new \Glpi\Inventory\Conf();
+        $this->assertTrue(
+            $conf->saveConf([
+                "enabled_inventory" => true,
+                'auth_required' => \Glpi\Inventory\Conf::CLIENT_CREDENTIALS,
+            ])
+        );
+        $DB->commit();
```

Comment:
I don't think it is a good idea to commit all changes in the transaction to the test DB.